nss-day-cohort-43-csharp/MVC-Silver-Snakes
# Bs uploading category to GitHub

# Description
Completed the functionality to view, add, edit, and delete categories.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Please check to ensure you can view, add, edit, and delete categories in the Category Management section.
# Checklist:
